WordPress/Classes/Services/BlogService.m

```diff
@@ -244,6 +249,60 @@ - (void)syncBlog:(Blog *)blog success:(void (^)())success failure:(void (^)(NSEr
     }
 }
 
+- (void)batchSyncBlog:(Blog *)blog
```

Comment:
Not to nitpick here, but this doesn't seem so much like batching as it is staggering. Thoughts on renaming?